cmd/extract-block-events/main.go

```diff
@@ -75,7 +75,7 @@ func run() int {
 	}
 	log = log.Level(level)
 
-	// Initialize the protocol state database we will use.
+	// Initialize the protocol state database.
```

Comment:
This is also a useless comment now. If we keep it, it should be more descriptive.

` Initialize the protocol state database used to extract blockchain data.` ...or something like that.